setup.py

```diff
@@ -25,7 +25,7 @@
 
 # Only include pytest-runner in setup_requires if we're invoking tests
 if {'pytest', 'test', 'ptr'}.intersection(sys.argv):
-    setup_requires = ['pytest-runner']
+    setup_requires = ['pytest-runner', 'fastparquet', 'pyarrow']
```

Comment:
Please revert this. `setup_requires` is for installing dependencies that are required for building the package (of which neither of these are).

Reply:
done